extensions/mssql/UPDATING_SQLTOOLSSERVICE.md

```diff
@@ -0,0 +1,14 @@
+# Updating SQL Tools Service
+
+This extension uses [SQL Tools Service](https://github.com/Microsoft/sqltoolsservice) to handle much of the direct interaction with the server, such as completions, language services and running queries.
+
+To update the version of SQL Tools Service that this extension uses follow these steps :
+
+1. Make your changes in the SQL Tools Service repo and get them merged in to main
+2. Wait for the next release to be published. The publish pipeline runs daily and the releases will be published [here](https://github.com/microsoft/sqltoolsservice/releases)
+3. Once the release is published and you've verified that it contains your changes then update [config.json](./config.json) and set the `version` field to the new version
```

Comment:
normally, updating the version is enough, there could be cases you also need to modify the file names, maybe we should mention it too? e.g.: https://github.com/microsoft/azuredatastudio/commit/2b2193b387a25f6f127d2532b78e40be9ee09b93#diff-329cc42d19e068edabab0b88e9d2a73ed2550dbd35936cfc10df4f28f16bcab7